### PR TITLE
Patch the logo in

### PR DESCRIPTION
**Pull request by ajyong**
Mmmm, style attribute hacks...

## Desktop
image

## Tablet
image

## Mobile
(Oh shoot, ~~the menu is broken~~ mobile doesn't have a menu, probably because we still have links outside the page)
image

### CODE REVIEW

**Comment by ajyong** [comment]
[ch223]

**Comment by MandyMeindersma** [comment]
lgtm

idk if you didn't want this merged.... sorry

**Comment by ajyong** [comment]
All good, honestly haven't used GitHub since they added the "Review" feature.  Mostly needed a second pair of eyes.  Thanks!

File: index.html [hunk 1]
```diff
@@ -66,7 +66,12 @@
       <!-- Intro -->
       <section id="intro" class="wrapper style1 fullscreen fade-up">
         <div class="inner">
-          <h1>CODEVID-19</h1>
+          <div class="image">
+            <img
+              src="/images/codevid-19_whitelogo.png"
+              style="max-width: 75%; margin: 0 1.5em 1em 0; top: 0.25em"
```
Comment (MarkBennett) [inline index.html:72]: We should have alt text here. Also for SEO it's good to keep the `<h1>` as Google, et al rely on it for understanding the page. We can use the `<header>` tag to flag this for an image.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

If you'd like to merge this PR I can update.

**Review by MarkBennett — COMMENTED** [review]
Just some comments. I'll make a new PR to addres..

File: index.html (hunk 1 above)
Comment (ajyong) [reply index.html:72]: Ah, glad there's a way to do this and I've learned something.  Will review your new PR soon™

**Pull request merged**